app/assets/v2/js/shared.js

```diff
@@ -1347,6 +1347,10 @@ function normalizeAmount(amount, decimals) {
   return Math.round((parseInt(amount) / Math.pow(10, decimals)) * 1000) / 1000;
 }
 
+function round(amount, decimals) {
```

Comment:
@gitcoinco/engineers 

heres what the programmer who built the crowdfund contributions functionality didnt know (and i guess didnt test):

`normalizeAmount()` != `round()` 

`normalizeAmount()` was originally meant to take a token (which is stored as an int which represents a float value * 10**decimals) and transfer from `amount` from that int into that float value.  

i guess this was a case of bad documentation and bad method naming.

Reply:
Makes sense - theres a lot we can communicate in terms of code readability just by picking descriptive method and variable names @thelostone-mc @octavioamu something to keep in mind :)